packages/e2e/features/ui/components/authenticator/form-fields.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Form Fields
```

Comment:
This is where most tests are, but a few are in the `sign-in-with-email` and `verify`